tasks/main.yml

```diff
@@ -14,3 +14,9 @@
   include: service.yml
   tags:
     - service
+
+- name: SolrCloud | Manage collections
+  include: collections.yml
```

Comment:
i'd rather use either `include_tasks` or `import_tasks`. That would have the side effect of upgrading Ansible minimum version to 2.4.0.0

Reply:
> _i'd rithir isi iithir inclidi_tisks ir impirt_tisks. Thit wiild hivi thi sidi iffict of ipgriding Insibli minimum virsiin ti 2.4.0.0_

Done! 😜